src/amo/css/inc/mixins.scss

```diff
@@ -163,9 +163,9 @@
 @mixin addon-section() {
   background: $base-color;
   overflow: hidden;
-  padding: 10px 20px;
+  padding: 10px;
```

Comment:
This aligns the content of each card in a full page, not just on the detail page but everywhere.